File: src/main/webapp/js/service/graduateinfoService.js
//服务层
app.service('graduateinfoService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../graduateinfo/findAll.do');		
	}
	//分页 
	this.findPage=function(page,rows){
		return $http.get('../graduateinfo/findPage?page='+page+'&rows='+rows);
	}
	//查询实体
	this.findOne=function(id){
		return $http.get('../graduateinfo/findOne?id='+id);
	}
	//增加 
	this.add=function(entity){
		return  $http.post('../graduateinfo/add',entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.post('../graduateinfo/update',entity );
	}
	//删除
	this.dele=function(ids){
		return $http.get('../graduateinfo/delete?ids='+ids);
	}
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('../graduateinfo/search?page='+page+"&rows="+rows, searchEntity);
	}    	
});

File: src/main/java/top/axbt/keshe/service/GraduateinfoService.java
package top.axbt.keshe.service;
import java.util.List;

import top.axbt.entity.PageResult;
import top.axbt.keshe.domain.TbGraduateinfo;

/**
 * 服务层接口
 * @author Administrator
 *
 */
public interface GraduateinfoService {

	/**
	 * 返回全部列表
	 * @return
	 */
	public List<TbGraduateinfo> findAll();
	
	
	/**
	 * 返回分页列表
	 * @return
	 */
	public PageResult findPage(int pageNum, int pageSize);
	
	
	/**
	 * 增加
	*/
	public void add(TbGraduateinfo graduateinfo);
	
	
	/**
	 * 修改
	 */
	public void update(TbGraduateinfo graduateinfo);
	

	/**
	 * 根据ID获取实体
	 * @param id
	 * @return
	 */
	public TbGraduateinfo findOne(Integer id);
	
	
	/**
	 * 批量删除
	 * @param ids
	 */
	public void delete(Integer[] ids);

	/**
	 * 分页
	 * @param pageNum 当前页 码
	 * @param pageSize 每页记录数
	 * @return
	 */
	public PageResult findPage(TbGraduateinfo graduateinfo, int pageNum, int pageSize);
	
}
